api/src/services/config_service.rs

```diff
@@ -45,6 +52,7 @@ pub struct ContainerConfig {
 pub struct JiraConfig {
     host: String,
     user: String,
+    // TODO: use secstr
```

Comment:
I guess that this definitely warrants an issue?

Reply:
That was a leftover... Now, it is fixed.